sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/ier12-integration.scheduler.ts

```diff
@@ -19,53 +22,36 @@ export class IER12IntegrationScheduler extends BaseScheduler<GeneratedDateQueueI
     super(schedulerQueue, queueService);
   }
 
-  /**
-   * To be removed once the method {@link process} is implemented.
-   * This method "hides" the {@link Process} decorator from the base class.
-   */
-  async processQueue(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
-  /**
-   * When implemented in a derived class, process the queue job.
-   * To be implemented.
-   */
-  protected async process(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
   /**
    * Identifies all the applications which are in assessment
    * for a particular institution and generate the request file.
    * @params job has generatedDate Date in which the assessment for
```

Comment:
Minor and not part of the PR. The comment must be `@param`

Reply:
Changed.